### PR TITLE
Add "OS font" patch

### PR DESCRIPTION
**Pull request by milyd**
Add "OS font" patch

### CODE REVIEW

**Review by banocean — COMMENTED** [review]
Since I can't test it, may I take a look at screenshots how it looks like before and after?

File: apply.js [hunk 1]
```diff
@@ -67,6 +67,13 @@ const patches = [
       css: ["hideHelpOnDashboard.css"],
     },
   },
+  {
+    name: "Custom font",
+    description: 'Applies default OS font',
```
Comment (banocean) [inline apply.js:72]: It's not even a sentence :sob:, pls change name and description to ones actually explaining what does it do

**Review by banocean — CHANGES_REQUESTED** [review]

**Comment by milyd** [comment]
Zrzut ekranu z 2024-08-31 18-48-30

The font is Roboto because i'm using Chrome on Ubuntu

**Comment by banocean** [comment]
> Zrzut ekranu z 2024-08-31 18-48-30
> 
> The font is Roboto because i'm using Chrome on Ubuntu

I asked for screenshots with this patch on and off OS this pr is targeting :roll_eyes:

**Comment by PanLiszka** [comment]
> Since I can't test it, may I take a look at screenshots how it looks like before and after?

Before:
image

After:
opera_IugTiNU0Ho

**Comment by PanLiszka** [comment]
> > Since I can't test it, may I take a look at screenshots how it looks like before and after?
> 
> Before: image
> 
> After: opera_IugTiNU0Ho

Doesn't change much on windows

**Comment by banocean** [comment]
> > Since I can't test it, may I take a look at screenshots how it looks like before and after?
> 
> Before: image
> 
> After: opera_IugTiNU0Ho

To be honest, the original one looks better. I opt for closing this pr

**Comment by Marioneq4958** [comment]
Why don't use font-family system-ui?

**Comment by banocean** [comment]
> Why don't use font-family system-ui?

it changes sizes of the element, you can tell that this site is better suited for the font they picked

**Pull request closed without merging**